src/components/Form.js

```diff
@@ -285,6 +285,7 @@ const Form = ({form}) => {
     <ProgressIndicator
       title={form.name}
       steps={form.steps}
+      hideNonApplicableSteps={form.hideNonApplicableSteps}
```

Comment:
Can you please add the new `hideNonApplicableSteps` prop to the mocks in `src/api-mocks/forms.js`?